docs/concepts/governance.md

```diff
@@ -41,9 +41,9 @@ sources={{
 - Voting power is calculated based on the current epoch's active stake of the proposer or voter's backing stake pool. In addition, the stake pool's lockup must be at least as long as the proposal's duration.
 - Verify proposals before voting. Ensure each proposal is linked to its source code, and if there is a corresponding AIP, the AIP is in the title and description.
 
-:::tip
-Each stake pool can be used to vote on each proposal exactly only one time.
-:::
+If you are a [staking pool](../../../concepts/staking.md) voter, follow the instructions for voting [here](https://aptos.dev/nodes/validator-node/voter/index/#steps-using-governance-ui).
+
+If you are a [delegated staker](../../../concepts/delegated-staking.md), follow the instructions for voting [here](https://aptos.dev/nodes/validator-node/voter/index/#steps-using-aptos-cli-1).
```

Comment:
These seem to have failed, I think that the `../../` system doesn't work quite well enough.

Let's use absolute links for now:
`[staking pool](/concepts/staking.md)`

Similarly for the other ones:
`[here](/nodes/validator-node/voter/index/#steps-using-governance-ui)`